src/app/privatechat-id/privatechat-id.component.html

```diff
@@ -129,7 +129,7 @@ <h3 class="rk-friend-name">{{ (friend | async)?.displayName }}</h3>
       <span class="material-icons remove-img" (click)="imgclear()">remove_circle</span>
         <img [src]="imgSrc" class="chat-img" alt="">
         <span class="message">{{ chat }}</span>
-        <span class="timestampout">Your image waiting...</span>
+        <span class="timestampout">Sending image...</span>
```

Comment:
Here if we add "sending image" they will think it's uploading yet. What's your solution?